### ISSUE TITLE
Detect loaded file type

### ISSUE THREAD

**Issue by ThaDafinser** [browscap/browscap-php#148]
After the issue here https://github.com/browscap/browscap/issues/907 i was thinking about splitting my detection and comparison for each downloadable file type, which would make sense i think.

My questions: 
Is there a way i can detect the loaded file type?
What is the idea/difference between `full` and `php` ?
 - Lite is for speed thats clear
 - just no device model/brand for a bit more speed?

I already did a comparison local 
![image](https://cloud.githubusercontent.com/assets/533017/13663366/5d498f14-e6a1-11e5-8549-3489feaf357a.png)

**Comment by asgrim** [comment]
@ThaDafinser I've created a new wiki page containing the answer to this, as it's a useful piece of information: https://github.com/browscap/browscap/wiki/Which-version-should-I-use%3F

Can I ask your permission to re-use that comparison chart on that page to demonstrate the difference please?

**Comment by ThaDafinser** [comment]
@asgrim sure you can use it :+1: 

if i'm right the `type` should be added like version/released here 
https://github.com/browscap/browscap-php/blob/master/src/Cache/BrowscapCacheInterface.php#L64-L76
```
[GJK_Browscap_Version]
Version=6012
Released=Thu, 04 Feb 2016 12:59:23 +0000
Format=php
Type=LITE
```

**Comment by asgrim** [comment]
Yup, the `Type` field will be one of `LITE`, `FULL` or blank (`Type=`), it's existed in the INI file for a while, but I guess it was just never exposed :)

**Comment by asgrim** [comment]
@ThaDafinser I've updated the wiki page with your comparison, thank you :)

**Comment by asgrim** [comment]
Closed by #149
